maap/Secrets.py

```diff
@@ -0,0 +1,129 @@
+import requests
+import logging
+import json
+from maap.utils import endpoints
+from maap.utils import requests_utils
+
+
+class Secrets:
```

Comment:
Could you add a section in the functional test to test adding, retrieving and deleting a secret? 
https://github.com/MAAP-Project/maap-py/blob/master/test/functional_test.py

Reply:
Yeah, this is next on my to-do.

Reply:
Implemented.